src/CoreBundle/Controller/DefaultController.php

```diff
@@ -217,4 +218,30 @@ public function unsubscribeMailConfirmAction($mail, $hash)
 
         return $this->render('account/unsubscribe_failure.html.twig');
     }
+
+    /**
+     * @Route("/feedback", name="feedback")
+     * @Security("has_role('ROLE_USER')")
+     */
+    public function feedbackAction(Request $request, Account $account)
+    {
+        $form = $this->createForm(FeedbackType::class,null,[
+            'action' => $this->generateUrl('feedback'),
+        ]);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $this->get('app.mailer.account')->sendCustomFeedbackToSystem($account, $this->getParameter('feedback_mail'), $form->getData()['message']);
+            return $this->render('feedback.html.twig', array(
+                'form' => $form->createView()
+            ));
+        }
+
+        return $this->render('feedback.html.twig', array(
+            'form' => $form->createView()
+        ));
+
+        return $this->redirectToRoute('dashboard');
```

Comment:
Unreachable statement.

In addition: Submitting the form will fail if `feedback_mail` is not valid. And `enable_feedback` is not checked. I guess you want to check that in a custom template to show a link to this route, but another if-block checking for `enable_feedback` will give a reason for this (currently unreachable) statement ;)